src/app/DomainPermissionsPage.qml

```diff
@@ -0,0 +1,290 @@
+            }
+
+            ViewItems.selectMode: selectMode
+
+            delegate: ListItem {
+                id: item
+                readonly property bool isCurrentItem: item.ListView.isCurrentItem
+                readonly property string domain: model.domain
+                height: isCurrentItem ? layout.height : units.gu(5)
+                color: isCurrentItem ? theme.palette.selected.base : theme.palette.normal.background
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: domainPermissionsListView.currentIndex = index
+                }
+
+                SlotsLayout {
+                    id: layout
+                    width: parent.width
+
+                    mainSlot:
+
+                        Column {
+
+                        spacing: units.gu(2)
+
+                        Row {
+                            spacing: units.gu(1.5)
+                            height: units.gu(1)
+                            width: parent.width
+
+                            Icon {
+                                visible: (model.permission === DomainPermissionsModel.NotSet)
+                                name: "dialog-question-symbolic"
+                                height: units.gu(2)
+                                width: height
+                            }
+
+                            IconWithColorOverlay {
+                                overlayColor: theme.palette.normal.positive
+                                visible: (model.permission === DomainPermissionsModel.Whitelisted)
+                                name: "emblem-default-symbolic"
+                                height: units.gu(2)
+                                width: height
+                            }
+
+                            IconWithColorOverlay {
+                                overlayColor: theme.palette.normal.negative
+                                visible: (model.permission === DomainPermissionsModel.Blocked)
+                                name: "action-unavailable-symbolic"
+                                height: units.gu(2)
+                                width: height
+                            }
+
+                            Label {
+                                text: model.domain
+                                font.bold: item.ListView.isCurrentItem
+                                color: theme.palette.normal.foregroundText
+                            }
+
+                            Label {
+                                visible: (model.requestedByDomain !== "")
+                                text:  "(→%1)".arg(model.requestedByDomain)
+                                color: theme.palette.normal.foregroundText
+                            }
+
+                        }
+
+                        ColumnLayout {
+                            visible: item.ListView.isCurrentItem
+                            CustomizedRadioButton {
```

Comment:
I'm unable to select any of these radio buttons upon creating a site. I created `example.com` while on a different site, then tried to select a radio button. No dice.

Reply:
uh, that is strange, can it be because I am on the edge channel ? Is the "Not Set" radio button selected after you create "example.com" ? For me if I click on the radio buttons or the texts "Blocked" or "Whitelisted", the selection changes.

Reply:
It happens only if: 
- there is no previous information stored (like used for the first time) 
- go to settings / domain permisions
- add a url
- tap on select

Reply:
@UniversalSuperBox @cibersheep  unfortunately I cannot reproduce that, could you maybe add a screenshot,or does the log say sth?
If you activate the checkbox "domain whitelist mode" (old version) / "only allow browseing to whitelisted websites" (new version), does morph properly ask for each new domain if it should be blocked or whitelisted ? Are these settings reflected in the domain permissions menu ?

Reply:
Hmm. I originally reproduced that on my Oneplus One across a few restarts of the browser and phone, but I can't do it any longer. Guess we'll keep watch for that one.

Reply:
It looks I cannot reproduce it anymore. I guess after rebooting a couple of times is gone. Mmmm....